src/shared/types/user.ts

```
@@ -1,4 +1,6 @@
 export type User = {
-  username: string;
-  email: string;
+  readonly id: String;
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)